website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,324 @@
+        label
+          strong {{$t('guildInformation')}}*
+        b-form-input(type="text", textarea :placeholder="$t('guildInformationPlaceHolder')", v-model="newGuild.guildInformation")
+
+      .form-group(style='position: relative;')
+        label
+          strong {{$t('categories')}}*
+        div.category-wrap(@click.prevent="toggleCategorySelect")
+          span.category-select(v-if='newGuild.categories.length === 0') {{$t('none')}}
+          .category-label(v-for='category in newGuild.categories') {{$t(categoriesHashByKey[category])}}
+        .category-box(v-if="showCategorySelect")
+          .form-check(
+            v-for="group in categoryOptions",
+            :key="group.key",
+          )
+            label.custom-control.custom-checkbox
+              input.custom-control-input(type="checkbox", :value='group.key' v-model="newGuild.categories")
+              span.custom-control-indicator
+              span.custom-control-description(v-once) {{ $t(group.label) }}
+          button.btn.btn-primary(@click.prevent="toggleCategorySelect") {{$t('close')}}
+
+      .form-group.text-center
+        div.item-with-icon
+          img(src="~assets/guilds/green-gem.svg")
+          span.count 2
+        button.btn.btn-primary.btn-md(v-if='!newGuild.id', :disabled='!newGuild.name || !newGuild.description') {{ $t('createGuild') }}
+        button.btn.btn-primary.btn-md(v-if='newGuild.id', :disabled='!newGuild.name || !newGuild.description') {{ $t('updateGuild') }}
+        div.gem-description
+          | {{ $t('guildGemCostInfo') }}
+</template>
+
+<style lang="scss" scoped>
+  @import '~client/assets/scss/colors.scss';
+
+  textarea {
+    height: 150px;
+  }
+
+  .description-count, .gem-description {
+    font-size: 12px;
+    line-height: 1.33;
+    text-align: center;
+    color: $gray-200;
+  }
+
+  .description-count {
+    text-align: right;
+  }
+
+  .gem-description {
+    margin-top: 1em;
+  }
+
+  .category-box {
+    padding: 1em;
+    max-width: 400px;
+    position: absolute;
+    top: -480px;
+    padding: 2em;
+    border-radius: 2px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.15), 0 1px 4px 0 rgba(26, 24, 29, 0.1);
+  }
+
+  .category-label {
+    min-width: 100px;
+    border-radius: 100px;
+    background-color: $gray-600;
+    padding: .5em;
+    display: inline-block;
+    margin-right: .5em;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 1.33;
+    text-align: center;
+    color: $gray-300;
+  }
+
+  .item-with-icon {
+    display: inline-block;
+
+    img {
+      height: 20px;
+      margin-right: .5em;
+    }
+
+    .count {
+      font-size: 14px;
+      font-weight: bold;
+      margin-right: 1em;
+      color: $green-10;
+    }
+  }
+
+  .description-count {
+    margin-top: 1em;
+  }
+
+  .category-select {
+    border-radius: 2px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
```

Comment:
all these shadows are black so you can do `rgba($black, 0.16)` (`rgba()` is a sass function)